packages/cli/src/ui/hooks/slashCommandProcessor.ts

```diff
@@ -740,15 +740,29 @@ export const useSlashCommandProcessor = (
                 // it.
                 chat.addHistory(item);
```

Comment:
This is the piece that adds the function call / response to the undisplayed chat history